### PR TITLE
AT _05.04_003 | <New item> Folder | Create Folder 

### PR DESCRIPTION
**Pull request by msnata**
https://trello.com/c/qN30qkpC

### CODE REVIEW

File: cypress/e2e/newItemFolder.cy.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/// <reference types="cypress"/>
+
+import createFolderName from '../fixtures/createFolderName.json'
+
+describe('newItemFolder', () => {
+    it('TC _05.04_003 Create Folder through "Create a job" link', () => {
```
Comment (DariaMas7) [inline cypress/e2e/newItemFolder.cy.js:6]: in it() we include AT numbers -name

File: cypress/e2e/newItemFolder.cy.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        cy.get("a[href='newJob']").click();
+        cy.get("#name").type(createFolderName.folderName);
+        cy.get("#j-add-item-type-nested-projects .j-item-options").contains("Folder").click();
```
Comment (DariaMas7) [inline cypress/e2e/newItemFolder.cy.js:9]: Folder should be moved to fixtures

**Review by DariaMas7 — COMMENTED** [review]
Hello. Its better to write your it() in an already existing file instead of creating a new one

File: cypress/e2e/newItemFolder.cy.js (hunk 2 above)
Comment (msnata) [reply cypress/e2e/newItemFolder.cy.js:9]: Вы имеете ввиду "Folder" в строке 9 cy.get("#j-add-item-type-nested-projects .j-item-options").contains("Folder").click(); ?
Если да, то подскажите пож в какой файл я могу положить? В тот, в котором создала имя??? Спасибо!

File: cypress/e2e/newItemFolder.cy.js (hunk 1 above)
Comment (msnata) [reply cypress/e2e/newItemFolder.cy.js]: Откорректировано

File: cypress/e2e/newItemFolder.cy.js (hunk 2 above)
Comment (msnata) [reply cypress/e2e/newItemFolder.cy.js]: Откорректировано

File: cypress/e2e/newItemFolder.cy.js (hunk 1 above)
Comment (msnata) [reply cypress/e2e/newItemFolder.cy.js]: Для US 05.04 файл не нашла. Для 05.07, 05.08 есть.   Если PR пройдет, можно будет смержить?

Comment (DariaMas7) [reply cypress/e2e/newItemFolder.cy.js]: So following your logic, the US which matches your AT should be US_02.01 | <Homepage> Create a job link

Comment (DariaMas7) [reply cypress/e2e/newItemFolder.cy.js]: Screenshot 2023-05-29 222827
Screenshot 2023-05-29 222804
Based on what I see US 05/04 in steps clearly states that the user clicks on NewItem and the description of US02/01 states that creating a job is located at a main panel. So there are two ways how a user may create a new job and the name of your commit includes <New item> and your code doesn't use this option

**Review by SvetlanaVet — APPROVED** [review]
it's ok

**Review by AlisaKapitanova — APPROVED** [review]

**Pull request merged**